packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html

```diff
@@ -1,5 +1,5 @@
 <div data-ui-view>
-    <header class="mb-3 oui-header">
+    <header class="mb-3 oui-header" data-ng-if="$ctrl.archive">
```

Comment:
Header is no longer relevant in other cases ?

Reply:
Its not relevant in `objects`. And to be used only in `cloud-archive`